.evg.yml

```diff
@@ -346,20 +347,23 @@ tasks:
           script: |
             set -e
             cd realm-cli
+            curl https://stedolan.github.io/jq/download/linux64/jq > jq && chmod +x jq
+            export VERSION=`cat package.json | ./jq '.version' | sed -e 's/^"//' -e 's/"$//'`
+
             if [ "${branch_name}" != "master" ]; then
               echo "skipping release: not on master branch"
               exit 0
             fi
-            if [ $(git tag -l "v${cli_version}") ]; then
+            if [ $(git tag -l "v$VERSION") ]; then
               echo "skipping release: tag already exists"
               exit 0
             fi
             echo "updating manifest"
-            ./realm-cli/contrib/manifest.js ${build_id} ${cli_version} manifest.json
-            echo "tagging release: v${cli_version}"
+            ./contrib/manifest.js ${build_id} $VERSION ../manifest.json
```

Comment:
...and this was the previous bug that would have blocked releasing `2.3.1`.  but once this merge goes into `master` and `baas` publishes its artifacts to `s3` for `469728`, `2.3.1` should be released at that point